### ISSUE TITLE
Feature request: color picker

### ISSUE THREAD

**Issue by marcingarbarczyk** [2sic/2sxc#1280]
**I'm submitting a ...**  
[x] feature request

**...about**   
[x] edit experience / UI
[x] admin experience UI
[x] Content Types or data management

**Current behavior**
There is no color picker data field.

**Expected behavior**
I think the color picker is a very useful function, good for UX

**Minimal reproduction of the problem with instructions**

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**

* **2sxc version(s):** 9.4.3  
* **Browser:** [all]
* **DNN:** [all]
* **Language:** [any/all]

**Anything you would like to add**

**Comment by iJungleboy** [comment]
I agree that this would be cool - it won't be on our priorities ATM, so it would need a community contribution. If anybody wants to pick this up, please keep in mind that

* no jQuery solutions (we're keeping our UI clean of jQuery)

**Comment by iJungleboy** [comment]
There isn't one in 2sxc, but now it's easy to create your own using custom input fields - check out the tutorials: https://2sxc.org/dnn-tutorials/en/razor/ui211/page
